### DISCUSSION TITLE
Please look over our pull requests!

### DISCUSSION THREAD

**Discussion by baelany** [beetbox/beets#4737] [General]
Hello! My name is Belany and my partner, Elizabeth, and I are part of a course at the University of Michigan EECS 481 (Software Engineering). The final project of this class involves contributing to an open-source github repo and we chose to beets since its mission piques both of our interests.

We recently submitted two pull requests (https://github.com/beetbox/beets/pull/4734 and https://github.com/beetbox/beets/pull/4730) addressing issues #4298 and #4693. We would appreciate it if you could look over these pull requests and give feedback / accept them into the master branch. For this class, we are rewarded a bit of extra credit if we can manage to push any type of pull request changes into the master branch of an open-source github repo. Our github usernames are baelany and elyang0214.

Thanks so much!

**Comment by RollingStar** [comment]
That's awesome! Were you aware of beets before looking for a project? How did you find out about it?

**Reply by baelany** [reply]
Thanks! We found out about beets since it is a project students in a previous semester have worked on.
